Scripts/Interfaces/ISetEditingContext.cs

```diff
@@ -0,0 +1,59 @@
+﻿#if UNITY_EDITOR
+using System;
+using System.Collections.Generic;
+
+namespace UnityEditor.Experimental.EditorVR
+{
+	public interface ISetEditingContext
+	{
+		/// <summary>
+		/// All of the available editing contexts
+		/// </summary>
+		List<IEditingContext> allContexts { set; }
```

Comment:
Probably should be a delegate such that it can be enumerated dynamically?  Also, probably should explicitly be IShellContextLaunchers or maybe even a IUnitySceneEditorSubContextLauncher?  Which would make a pretty nice plugin system where you can have SubContexts that are marked for use in your Scene Editor rather than just all of them at all times.